### PR TITLE
Nano-S fixes for v0.14

### PR DESCRIPTION
**Pull request by cslashm**
+ New management of destination keys and their verification by user
+ next V10 protocol support
+ some debug fixes

### CODE REVIEW

File: src/device/device_ledger.cpp [hunk 1]
```diff
@@ -1479,20 +1489,31 @@ namespace hw {
             offset+=32;
           } else {
             // dummy: is_subaddress Aout Bout AKout
-            offset += 1+32*3;
+            offset += 2+32*3;
```
Comment (moneromooo-monero) [inline src/device/device_ledger.cpp]: Why is this change for ?

Comment (cslashm) [reply src/device/device_ledger.cpp]: one more byte for `is_change_address` in the `if` counterpart ,   comment was not updated...

Comment (moneromooo-monero) [reply src/device/device_ledger.cpp]: Hmm, OK, I don't see a code change to add that extra byte, that's what I was getting at.

**Comment by SamsungGalaxyPlayer** [comment]
Will this be for 0.14 or 0.14.1?

**Comment by cslashm** [comment]
need to be for 0.14, else Ledger HW will wont work

**Comment by moneromooo-monero** [comment]
Oh, thanks, I'd forgot about this. Can someone who has a ledger please test this ?

**Comment by moneromooo-monero** [comment]
What do you have left to do on this to lose the DO NOT MERGE part ?

**Comment by cslashm** [comment]
I have to check tow more things to ensure a "smooth" update between 0.13->0.14 with next Ledger app 1.2.0. 

It will be done this week. Then I will remove the "do not merge."

**Comment by moneromooo-monero** [comment]
Since you removed the do not merge now, can I assume you've tested and it's good to go ?

**Comment by cslashm** [comment]
I made:
 - A TX with app 1.2.0 and client 0.13.4 over stagenet: OK
- A TX with app 1.2.0 and client tweaked-master over testnet with protocol V10: OK

Seems good so.

**Comment by cslashm** [comment]
When the testnet will move to v10?

**Comment by moneromooo-monero** [comment]
Probably a day or two after everything's ready. Everything here means just the final PoW.

**Comment by lacksfish** [comment]
> Oh, thanks, I'd forgot about this. Can someone who has a ledger please test this ?

@moneromooo-monero I can give it a spin on my testing Ledger tomorrow. Will report back.

**Pull request merged**